webapp/src/components/Celebrities.vue

```diff
@@ -204,26 +214,53 @@
         // initialize lists of boxes and markers to be drawn
         var boxMap = new Map();
         var markers = [];
-        var es_data = this.elasticsearch_data
+        var es_data = this.elasticsearch_data;
+        var i=0;
```

Comment:
It isn't super clear what this variable is doing. I see you're updating the value after each iteration in the 'boxMap.set(i++, [boxinfo])', but this probably wouldn't be clear to someone right off the bat.

Reply:
I've added a comment to describe how we're using the boxMap key in another branch which committed in the next sprint.